github3/repos/repo.py

```diff
@@ -687,14 +687,15 @@ def create_commit(self, message, tree, parents, author=None,
         return self._instance_or_null(Commit, json)
 
     @requires_auth
-    def create_deployment(self, ref, force=False, payload='',
+    def create_deployment(self, ref, required_contexts=[], payload='',
```

Comment:
Even though we do no munging of parameter values, I would rather use None and then coerce it to an empty list inside the method. Using mutable default parameters is dangerous if we ever update this to munge `required_contexts`.